src/server/scripts/Kalimdor/zone_orgrimmar.cpp

```diff
@@ -263,8 +264,229 @@ class npc_thrall_warchief : public CreatureScript
 
 };
 
```

Comment:
Sure, either like that or by just 2 slashes ( // ) in front of the text (more commonly used for commenting in 1 line).
/* and */ is more often used for multiple lines or a paragraph, like the Copyright text at the start of the files.